amplify/backend/function/getUserNotificationsHMAC/src/index.js

```diff
@@ -0,0 +1,29 @@
+const Base64 = require('crypto-js/enc-base64');
+const hmacSHA256 = require('crypto-js/hmac-sha256');
+
+let magicbellApiSecret = process.env.MAGICBELL_API_SECRET;
+
+const setEnvVariables = async () => {
+  const ENV = process.env.ENV;
+
+  if (ENV === 'qa' || ENV === 'prod') {
+    const { getParams } = require('/opt/nodejs/getParams');
+    [magicbellApiSecret] = await getParams(['magicbellApiSecret']);
+  }
+};
+
+exports.handler = async (event) => {
+  try {
+    await setEnvVariables();
+  } catch (err) {
+    throw new Error('Unable to set environment variables. Reason:', err);
+  }
+
+  const checksummedWalletAddress = event.request.headers['x-wallet-address'];
```

Comment:
I took this pattern from the bridge lambdas. Is it safe to assume that the wallet address from the headers has already been checksummed, or do we need to pass it through ethers.utils.getAddress?

Reply:
The issue here is that I don't really trust where the address is coming from. It might be checksumed or it might be not

So I'd run it through `getAddress` just to be sure